<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseDialog</class>
 <widget class="QDialog" name="DatabaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Manager</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Meta</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Coordinate at the begin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Autor name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Character Tileset</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="AutotNamelineEdit_">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="tilesetList">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="xCooCharacterTilesetSpinBox">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Character coordinates in tileset</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="yCooCharacterTilesetSpinBox">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>80</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="xCooBeginSpinBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="yCooBeginSpinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Sons</string>
    </attribute>
    <widget class="QListWidget" name="audioList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>161</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Audio list</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Player</string>
     </property>
    </widget>
    <widget class="QPushButton" name="playSoundButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>87</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Play Sound</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopSoundButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>80</y>
       <width>87</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop sound</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Textes</string>
    </attribute>
    <widget class="QListWidget" name="textList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>161</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textContent">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>256</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>56</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Text list</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Text content</string>
     </property>
    </widget>
    <widget class="QToolButton" name="addTextToolButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>../resources/icons/icone-plus.png</normalon>
      </iconset>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DatabaseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DatabaseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
